### PR TITLE
[terra-functional-testing] Remove cucumber / jasmine references

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

Remove cucumber / jasmine references

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

File: packages/terra-functional-testing/CHANGELOG.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ## Unreleased
 
-* Initial stable release
+* Initial stable release.
```
Comment (mjhenkes) [inline packages/terra-functional-testing/CHANGELOG.md]: Now i think you're just messing with me

**Comment by mjhenkes** [comment]
Can you remove any dependencies with this change?

**Comment by emilyrohrbough** [comment]
It doesn't look like it. It seems the existing tests were all written and testing mocha but didn't test cucumber. Jasmine uses the same hooks that mocha does so they are 1:1.

**Review by mjhenkes — APPROVED** [review]

**Review by yuderekyu — APPROVED** [review]
just curious, why were the params updated with an underscore?

**Comment by emilyrohrbough** [comment]
@yuderekyu I've seen this with parameters that are unused. It'd doesn't really matter. We could remove a lot of these, but I didn't want to clean house till we've finalized out changes, otherwise we'd have to search through the wdio docs to find these parameter values.

**Pull request merged**